onvm/onvm_nflib/onvm_common.h

```diff
@@ -67,14 +67,14 @@
 
 #define PACKET_READ_SIZE ((uint16_t)32)
 
-#define ONVM_ENABLE_SHARED_CPU_DEFAULT 0  // default value for shared cpu logic, if true NFs sleep while waiting for packets
+#define ONVM_NF_CORE_SHARING_DEFAULT 0  // default value for shared core logic, if true NFs sleep while waiting for packets
```

Comment:
Can we change the comment to `// default for shared core logic, if true NFs sleep while waiting for packets`. Linter would stop complaining, right now we're at 123 characters

Reply:
Yeah good point, will fix